Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs

```diff
@@ -159,10 +169,24 @@ private void AddSmiteVerbs(GetVerbsEvent<Verb> args)
                 _polymorphableSystem.PolymorphEntity(args.Target, "AdminMonkeySmite");
             },
             Impact = LogImpact.Extreme,
-            Message = "Monkey mode.",
+            Message = Loc.GetString("admin-smite-monkeyify-description")
         };
         args.Verbs.Add(monkey);
 
+        Verb disposalBin = new()
+        {
+            Text = "Garbage Can",
```

Comment:
Text should be localised as well right.

Reply:
No, intentionally left unlocalized as the only way to see the text in-game is to use `invokeverb`, which is for automation/adminbus, not end users.